packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+import { injectIntl } from 'react-intl';
 import { Consumer } from 'xfc';
+import classNames from 'classnames/bind';
+import styles from './ProviderIframe.module.scss';
```

Comment:
Nit: for readability, let's group these as 3rd party, terra and then internal imports:
```suggestion
import React from 'react';
import { injectIntl } from 'react-intl';
import PropTypes from 'prop-types';
import classNames from 'classnames/bind';

import { Consumer } from 'xfc';
import VisuallyHiddenText from 'terra-visually-hidden-text';

import styles from './ProviderIframe.module.scss';
```

Reply:
Updated here: https://github.com/cerner/terra-framework/pull/1804/commits/30f50f9daf5e5969fb40d1e52de1aac0bdb0d18c